spotbugs/src/main/java/edu/umd/cs/findbugs/BugRanker.java

```diff
@@ -90,7 +93,7 @@
     /** Minimum value for user visible ranks (most relevant) */
     public static final int VISIBLE_RANK_MIN = 1;
 
-    static final boolean PLUGIN_DEBUG = Boolean.getBoolean("bugranker.plugin.debug");
+    private static final Logger LOG = LoggerFactory.getLogger(BugRanker.class);
```

Comment:
Note that you can use `LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());` to get a class-agnostic logger statement, easier to copy and paste.